playwright/e2e/threads/threads.spec.ts

```diff
@@ -0,0 +1,482 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { SettingLevel } from "../../../src/settings/SettingLevel";
+import { Layout } from "../../../src/settings/enums/Layout";
+import { test, expect } from "../../element-web-test";
+
+test.describe("Threads", () => {
+    test.use({
+        displayName: "Tom",
+        botCreateOpts: {
+            displayName: "BotBob",
+            autoAcceptInvites: true,
+        },
+    });
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto("/");
+        await page.evaluate(() => {
+            window.localStorage.setItem("mx_lhs_size", "0"); // Collapse left panel for these tests
+        });
+    });
+
+    // Flaky: https://github.com/vector-im/element-web/issues/26452
+    test.skip("should be usable for a conversation", async ({ page, app, bot }) => {
+        const roomId = await app.client.createRoom({});
+        await app.client.inviteUser(roomId, bot.credentials.userId);
+        await bot.joinRoom(roomId);
+        await page.goto("/#/room/" + roomId);
+
+        // Around 200 characters
+        const MessageLong =
+            "Hello there. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt " +
+            "ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi";
+
+        const ThreadViewGroupSpacingStart = "56px"; // --ThreadView_group_spacing-start
+        // Exclude timestamp and read marker from snapshots
+        const mask = [page.locator(".mx_MessageTimestamp"), page.locator(".mx_MessagePanel_myReadMarker")];
+
+        const roomViewLocator = page.locator(".mx_RoomView_body");
+        // User sends message
+        const textbox = roomViewLocator.getByRole("textbox", { name: "Send a message…" });
+        await textbox.fill("Hello Mr. Bot");
+        await textbox.press("Enter");
+
+        // Wait for message to send, get its ID and save as @threadId
+        const threadId = await roomViewLocator
+            .locator(".mx_EventTile[data-scroll-tokens]")
+            .filter({ hasText: "Hello Mr. Bot" })
+            .getAttribute("data-scroll-tokens");
+
+        // Bot starts thread
+        await bot.evaluate(
+            (bot, { roomId, threadId, MessageLong }) => {
+                return bot.sendMessage(roomId, threadId, {
+                    // Send a message long enough to be wrapped to check if avatars inside the ReadReceiptGroup are visible
+                    body: MessageLong,
+                    msgtype: "m.text",
+                });
+            },
+            { roomId, threadId, MessageLong },
+        );
```

Comment:
Why not update `sendMessage` to allow taking a threadId or use `bot.sendEvent` which already can?